test/integration/maincluster_test.go

```diff
@@ -893,6 +898,8 @@ experimental:
     enabled: true
   clusterAutoscalerSupport:
     enabled: true
+  clusterTLSBootstrap:
```

Comment:
@mumoshu I actually never gave much thought to the configuration name. I guess `tlsBootstrap` is better. 👍